.github/ALLOWLIST

```diff
@@ -14,6 +14,7 @@
 #
 
 cmd/configschema
+confmap/provider/s3mapprovider
```

Comment:
Any new component or folder must have an owner. You should add yourself and @Aneurysm9  if he is sponsoring

```suggestion
confmap/provider/s3mapprovider
```

Reply:
Fixed.